src/VirtualizingListBox/VirtualizingListBox.vala

```diff
@@ -769,6 +783,10 @@ public class VirtualizingListBox : Gtk.Container, Gtk.Scrollable {
         return null;
     }
 
+    protected void select_row (VirtualizingListBoxRow row) {
+        select_row_internal (row);
+    }
+
```

Comment:
Can't we just adjust the visibility of `select_row_internal`? This way we don't have to add another method which is doing the same thing - or am I missing something here?